new_app_setup.py

```diff
@@ -0,0 +1,266 @@
+
+    parser.add_argument('-LOU', '--list_of_users',
+                        required=False,
+                        help='A list of the users for this team.')
+
+    parser.add_argument('-RAM', '--RAM', type=int, required=False,
+                        help='How many gigabytes of RAM your team is '
+                        'requesting, the deault is 2G.')
+
+    parser.add_argument('-CPU', '--CPU', type=int, required=False,
+                        help="How much CPU your team is requesting, the "
+                        "default is one.")
+
+    args = parser.parse_args()
+
+    team_name = args.team_name.replace(" ", "-").replace("_", "-").lower()
+
+    namespace_filename = team_name+"_namespace.yml"
+
+    pods_filename = team_name+"_pods.yml"
+
+    create_kubernetes_namespace_yml(namespace_filename, team_name)
+
+    create_kubernetes(namespace_filename, team_name, "namespace")
+
+    if args.RAM is not None:
+        ram_requested = args.RAM
+    else:
+        ram_requested = '2G'  # default
+
+    if args.CPU is not None:
+        cpu_requested = args.CPU
+    else:
+        cpu_requested = '1'  # default
+
+    create_kubernetes_pods_yml(pods_filename, team_name, ram_requested,
+                               cpu_requested)
+
+    create_kubernetes(pods_filename, team_name, "pods")
+
+    config_file = get_congif_yml_file(team_name)
+
+    path_to_congif_file = os.path.join(base_path, config_file)
+
+    send_config_email_to_users(team_name, args.group_user, config_file)
+
+    if args.list_of_users is not None:
+        send_config_email_to_users(team_name, args.list_of_users, config_file)
+
+
+def create_kubernetes_namespace_yml(namespace_filename, team_name):
+
+    tag = "Hack_For_A_Cause_2019"
+
+    yml_file_kubernetes_data = dict(
+
+        kind='Namespace',
+        apiVersion='v1',
+        metadata=dict(
+            name=team_name,
+            labels=dict(
+                name=tag
+                )
+            )
+        )
+
+    with open(namespace_filename, 'w') as outfile:
+        yaml.dump(yml_file_kubernetes_data, outfile, default_flow_style=False)
+
+    print("The YAML file for {} was created and was "
+          "saved as {}".format(team_name, namespace_filename))
+
+    return yml_file_kubernetes_data
+
+
+def create_kubernetes_pods_yml(pods_filename, team_name, ram_requested,
+                               cpu_requested):
+    yml_file_kubernetes_data = dict(
+        kind='Pod',
+        apiVersion='v1',
+        metadata=dict(
+            name=team_name,
+            namespace=team_name,
+            ),
+        spec=dict(
+                containers=dict(
+                    name=team_name,  # QUESTION HERE -> see below
+                    resources=dict(
+                        limits=dict(
+                            cpu="1",
+                            memory="2G"
+                        ),
+                        requests=dict(
```

Comment:
so here the pods are requesting to *use* `cpu_requested` and `ram_requested` -- pods are the apps that run. What we want to do is create the **limits** on how much all the pods can use but leave it up to the teams to decide how many pods to run, what each pod does, and how much of the available resources each pod should use.